ipynb/primary-processing/2-filter_sequences_for_blooms.md

```diff
@@ -0,0 +1,102 @@
+Early in the American Gut Project, it was observed that some organisms bloomed likely as a result of increased shipping time and delay between when samples were collected and when they were put on ice (more detail can be found [here](http://americangut.org/?page_id=277)). The purpose of this notebook is to apply the filter developed in order to bioinformatically subtract these observed bloom sequences from fecal samples. It is important to apply this filter when combining data with the American Gut as to remove a potential study-effect bias as all fecal data in the American Gut has had this filter applied. The specific steps covered are:
+
+* Filter demultiplexed sequence data to only fecal samples
+* Determine what sequences in the fecal samples recruit to the observed bloom sequences
+* Remove the recruited bloom sequences from the demultiplexed sequence data
+* Trim the filtered demultiplexed data (to reduce study bias when combining with short reads, such as the data in [Yatsunenko et al 2012](http://www.nature.com/nature/journal/v486/n7402/abs/nature11053.html))
```

Comment:
Yes, saw that. Perhaps adding to the main description. Note that it talks about the bloom filtering but not about the length.